pkg/daemon/ceph/agent/flexvolume/controller.go

```diff
@@ -197,6 +197,27 @@ func (c *Controller) Attach(attachOpts AttachOptions, devicePath *string) error
 	return nil
 }
 
+// Resize RBD image
+func (c *Controller) Resize(expandArgs ExpandArgs, _ *struct{}) error {
+	expandOpts := expandArgs.ExpandOptions
+	sizeInMb := expandArgs.Size / 1024 / 1024
```

Comment:
Use https://github.com/rook/rook/blob/master/pkg/util/display/bytes.go#L62

Reply:
I haven't seen this method :) Ok, I will change it.